File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/GameFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.game.Game;

public class GameFilterValidator extends FilterValidator<Game> {

    public GameFilterValidator(CustomValidator<Game> validator) {
        super(Game.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/GameFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.game.Game
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class GameFilterValidator(validator: Validator<Game>) : FilterValidator<Game>(Game::class, validator)

